src/activity-functions-function-notation.xml

```diff
@@ -105,7 +106,8 @@
 
                     <p>Without knowing anything about function notation, you can still use a language you already know, English, to describe information about the train.</p>
 
-                    <p>For instance:<ul>
+                    <p>For instance:</p>
+                    <ul>
```

Comment:
<p>For instance:
                    <ul>
                        <li>
                            <p>The train can only be in one place at any particular time.</p>
                        </li>
                        <li>
                            <p>As time passes the train gets farther from downtown.</p>
                        </li>
                        <li>
                            <p>The speed of the train changes during its trip.</p>
                        </li>
                    </ul>
                    </p>